### PR TITLE
fix(docs-infra): switch code font from `Droid Sans Mono` to `Roboto Mono`

### PR DESCRIPTION
**Pull request by eneajaho**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] angular.io application / infrastructure changes

## What is the current behavior?

Issue Number: #40666 

## What is the new behavior?
Font family changed to Roboto Mono, monospace

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by eneajaho** [comment]
@googlebot I signed it!

**Comment by eneajaho** [comment]
@gkalpak should I also change the font-size from 90% to 100% (just will remove font-size style)? 

Before: 
image

After: 
image

File: aio/src/index.html [hunk 1]
```diff
@@ -26,7 +26,6 @@
 
   <!-- NOTE: These need to be kept in sync with `ngsw-config.json`. -->
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap">
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Droid+Sans+Mono&display=swap">
```
Comment (gkalpak) [inline aio/src/index.html:29]: We should load `Roboto Mono` instead (it is a different font than `Roboto`).

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx, @eneajaho 👍

I left a comment. Could you also squash the commits into one and use a commit message that more closely follows our [commit message guidelines](https://github.com/angular/angular/blob/1d9e56eba06958e37ef8062ae7b65afa85f90632/CONTRIBUTING.md#commit)? For example:

> fix(docs-infra): switch code font from \`Droid Sans Mono\` to \`Roboto Mono\`
>
> \# Briefly explain why we are making this change.
>
> Fixes #40666

**Comment by gkalpak** [comment]
You do not need to change the font-size. We are working on some changes to improve readability of the docs and part of that is increasing the font-sizes: #40427

File: aio/src/index.html (hunk 1 above)
Comment (eneajaho) [reply aio/src/index.html:29]: Done!

**Review by gkalpak — APPROVED** [review]
One minor suggestion. Otherwise lgtm :+1:

File: aio/src/index.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
...
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300;400;500&display=swap">
```
[the remark this answers is not in the document]
Comment (eneajaho) [reply aio/src/index.html:29]: Done.

**Pull request closed without merging**